src/main/java/filewalker/DistinguishFilesFromDirectories_3.java

```diff
@@ -0,0 +1,42 @@
+package filewalker;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DistinguishFilesFromDirectories_3 {
+
+    public static void main(String[] args) {
+        String aa = Filewalker.getPath();
+        String path = Filewalker.getPath();
+        List<String> distinguishedFileList = distinguishFileFromDirectory(path);
+        printTable(distinguishedFileList);
+
+    }
+
+    static List<String> distinguishFileFromDirectory(String path){
+        File file = new File(path);
+        List<String> distinguishedFileList = new ArrayList<>();
+
+        if(file.exists()){
+            File[] fileList = file.listFiles();
+            for(int i = 0;  i < fileList.length; i++) {
```

Comment:
Staraj korzystać się ze Stream API, zamiast podstawowych pętli, poprawi to czytelność kodu